epix/event_separation.py

```diff
@@ -0,0 +1,107 @@
+from scipy import interpolate
+import numpy as np
+
+def clean_separation(n_events, MaxDelay):
+    """
+    Function to generate evenly spaced event times.
+
+    Args:
+        n_events (int): Number of events 
+        MaxDelay (float): Time difference between events. Should be large enought to 
+            prevent event pile-up.
+
+    Returns:
+        dt (numpy.array): Array containing the start times of the events.
+    """
+    
+    dt = np.arange(0, n_events)+np.arange(0, n_events)/10
+    dt *= MaxDelay
+
+    return dt
+
+def times_from_fixed_rate(rate, n_events):
+    """
+    Function to generate event times with a fixed rate.
+
+    The event times are drawn from a uniform distribution.
+    For higher rates pile-up is possible. The normalization
+    is achived by a variable overall simulation length in time.
+
+    !Rate normalization is only valid for one simualtion job!
+
+    Args:
+        rate (int or float): Mean event rate in Hz. 
+        n_events (int): Number of events
+
+    Returns:
+        event_times (numpy.array): Array containing the start times of the events.
+    """
+    
+    simulation_time = n_events/np.float(rate) #seconds
```

Comment:
Why np.float?

Reply:
Good catch. Should not be necessary here.

Reply:
Done with https://github.com/XENONnT/epix/pull/6/commits/b6b99f31c212d938a63bedf23130db5a488e8f02.